### PR TITLE
Added LIS_OUTCOME_SERVICE_URL to launch call.

### PR DESCRIPTION
**Pull request by julien-sebire**
Requires https://github.com/oat-sa/tao-core/pull/2270
Actually does what the NEX-319 ticket asked for.

### CODE REVIEW

File: model/delivery/container/LtiDeliveryContainer.php [hunk 1]
```diff
@@ -55,16 +57,20 @@ public function getExecutionContainer(DeliveryExecution $execution)
...
-            'launch_presentation_return_url' => $returnUrl,
-            'lis_result_sourcedid' => $execution->getIdentifier(),
+            LtiLaunchData::LTI_MESSAGE_TYPE => 'basic-lti-launch-request',
+            LtiLaunchData::LTI_VERSION => 'LTI-1p0',
+            LtiLaunchData::RESOURCE_LINK_ID => $execution->getIdentifier(),
+            LtiLaunchData::USER_ID => $this->getServiceLocator()->get(SessionService::SERVICE_ID)->getCurrentUser()->getIdentifier(),
+            LtiLaunchData::ROLES => 'Learner',
+            LtiLaunchData::LAUNCH_PRESENTATION_RETURN_URL => $returnUrl,
+            LtiLaunchData::LIS_RESULT_SOURCEDID => $execution->getIdentifier(),
+            LtiLaunchData::LIS_OUTCOME_SERVICE_URL => $outcomeServiceUrl,
```
Comment (siwane) [inline model/delivery/container/LtiDeliveryContainer.php]: This constant comes with taoLti : 10.5.1, please update dependency to include this verison

**Review by siwane — CHANGES_REQUESTED** [review]

File: model/delivery/container/LtiDeliveryContainer.php (hunk 1 above)
Comment (julien-sebire) [reply model/delivery/container/LtiDeliveryContainer.php]: Fixed.

**Review by boajer — APPROVED** [review]
Nice job!

**Comment by boajer** [comment]
@mike-ionut-mihai-sandu-tao  @julien-sebire  I've merged develop into this branch, looks like there are almost no chages left. 
Is it still needed ?

**Review by mike-ionut-mihai-sandu-tao — COMMENTED** [review]
let's please check if this is still required and either merge or close this pr, also let's check the version from manifest

**Pull request closed without merging**